### PR TITLE
MUI/ - messing with the MUI button

### PR DESCRIPTION
**Pull request by RChandler234**
## Changes

Testing out MUI with a couple of components starting with the Action-Button.

## Notes

Planning to continue transitioning files from react-bootstrap to MUI using this PR, wanted to get it up for visibility.

## Screenshots (if applicable)

MUI Button:
image
Old Button:
image

## To Do

Transitioning the rest of the react-bootstrap stuff, will compile a list here.

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://github.com/Northeastern-Electric-Racing/FinishLine/blob/develop/docs/ContributorGuide.md) and reach out to your squad if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors
- [ ] No newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (if applicable)
- [ ] Remove any not-applicable sections
- [ ] Assign the PR to yourself
- [ ] PR is linked to the ticket
- [ ] No `package-lock.json` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack

Closes # (issue #)

### CODE REVIEW

**Comment by jamescd18** [comment]
Are you intentionally merging to develop?

File: src/frontend/src/components/ActionButton.tsx [hunk 1]
```diff
@@ -26,12 +23,13 @@ interface ActionButtonProps {
 const ActionButton: React.FC<ActionButtonProps> = ({ link, icon, text }) => {
   return (
     <Link className={'row py-auto px-3 '} to={link} style={{ textDecoration: 'none' }}>
-      <Button>
+      <Button
+        variant="contained"
+        style={{ backgroundColor: '#ef4345', textTransform: 'none', fontSize: '15px' }}
+      >
         <div style={styles.button}>
           <FontAwesomeIcon className="mr-2 my-auto" icon={icon} size="1x" color="white" />
```
Comment (jamescd18) [inline src/frontend/src/components/ActionButton.tsx:31]: MUI also has its own way of doing buttons with icons

**Comment by RChandler234** [comment]
> Are you intentionally merging to develop?

I just did that to open the PR so ppl could look at it, we def don't wanna merge to develop til the whole thing is done

**Comment by jamescd18** [comment]
Cool makes sense

**Comment by anthonybernardi** [comment]
see https://github.com/Northeastern-Electric-Racing/FinishLine/pull/174

**Pull request closed without merging**